app/views/placements/schools/placements/build/check_your_answers.html.erb

```diff
@@ -30,10 +30,18 @@
 
           <% summary_list.with_row do |row| %>
             <% row.with_key(text: t(".subject")) %>
-            <% row.with_value(text: @placement.subjects.map(&:name).to_sentence) %>
+            <% row.with_value(text: @placement.subject.name) %>
             <% row.with_action(text: t(".change"), href: add_subject_placements_school_placement_build_index_path(@school, :add_subject), visually_hidden_text: t(".add_subject")) %>
           <% end %>
 
+          <% if @placement.subject_has_child_subjects? %>
```

Comment:
Would this `if` statement be more appropriate as:

```suggestion
          <% if @placement.has_additional_subjects? %>
```

in other words

```suggestion
          <% if @placement.additional_subjects.present? %>
```

i.e. checking whether additional subjects are present on the placement, rather than if the subject has children?

It's not a strong opinion, but somehow it feels more semantically correct.

Reply:
We talked about how this saves us from doing more data massaging in the controller. Since we don't explicitly clear the `additional_subject_ids` from the session if the user changes the `subject_id` from MFL to something else.

So it seems fair enough to leave this as is for now. It's the simplest way to catch that edge case without adding extra stuff into the controller.